CactbotOverlay/FateWatcherIntl.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Machina.FFXIV;
+using Machina.FFXIV.Headers;
+
+namespace Cactbot {
+  class FateWatcherIntl : FateWatcher {
+
+    public FateWatcherIntl(CactbotEventSource client) : base(client) {
+      monitor = new FFXIVNetworkMonitor {
+        MessageReceived = (string connection, long epoch, byte[] message) => MessageReceived(message)
+      };
```

Comment:
Why do you spin up an additional network monitor? This way every packet will be decoded twice: Once by the FFXIV plugin and once by Cactbot. Also, the list of active connections, etc., all will be duplicated.
The FFXIV plugin raises an event for each received packet which makes all of this unnecessary. [OverlayPlugin even provides a handy wrapper](https://github.com/ngld/OverlayPlugin/blob/master/OverlayPlugin.Core/FFXIVRepository.cs#L155).